test/e2e/page-objects/pages/confirmations/redesign/confirmation.ts

```diff
@@ -91,6 +97,34 @@ class Confirmation {
       throw e;
     }
   }
+
+  async checkLoadingSpinner(): Promise<void> {
+    await this.driver.assertElementNotPresent(this.loadingSpinner);
+  }
+
+  async checkBannerAlertIsNotPresent(): Promise<void> {
+    await this.driver.assertElementNotPresent(this.bannerAlert);
+  }
+
+  async validateBannerAlert({
```

Comment:
If this isn't triggering anything, should this also be a `check_` for consistency?